_rules/sequentially-focusable-element-has-visible-focus-oj04fd.md

```diff
@@ -34,7 +34,7 @@ This rule assumes that documents with only one focusable element automatically s
 
 ## Accessibility Support
 
-There are no major accessibility support issues known for this rule.
+There are There are no accessibility support issues known..
```

Comment:
Also not sure why some (like here) are in straight up txt and some (like [this one](https://github.com/act-rules/act-rules.github.io/pull/1945/files#diff-579ce0c887300ad57b274db4b0c63bfbcdbb8e74c9aaebdf54ad0b326fbb8cd3R63)) are in italics/emphasised text.

Reply:
@giacomo-petri  I'd say I included updates to those files as well. Feel free to raise it if I did not.

Reply:
I'd like for them to be italic personally. I think that sets them apart from the ones where there is information. Slightly easier to scan the page that way.